NamedNode is not generic

I'm getting the following error because type declarations use `NamedNode` with a type parameter.

```
node_modules/@tpluscode/rdf-string/lib/nquads.d.ts:18:132 - error TS2315: Type 'NamedNode' is not generic.

18 export declare const nquads: (strings: TemplateStringsArray, ...values: Value<TemplateResult<NQuadsOptions>, BlankNode | Literal | 
NamedNode<string>>[]) => TemplateResult<NQuadsOptions>;
```

The `.d.ts` files get generated like that despite the fact that the ts code contains no such argument. It seems to be a bug in typescript that is solved by updating typescript.

--- a/package.json
+++ b/package.json
@@ -54,7 +54,7 @@
     "standard": "^14.3.1",
     "string-to-stream": "^3.0.1",
     "typedoc": "^0.17.3",
-    "typescript": "^3.8.3"
+    "typescript": "^3.9.5"
   },
   "repository": {
     "type": "git",

